codespell_lib/data/dictionary.txt

```diff
@@ -27423,6 +27426,11 @@ trabsform->transform
 traceablity->traceability
 trackign->tracking
 trackling->tracking
+tracsode->transcode
+tracsoded->transcoded
+tracsoder->transcoder
```

Comment:
Just out of curiosity, how is this PR passing the CI checks?
http://app.aspell.net/lookup?dict=en_US-large;words=transcode%0D%0Atranscoded%0D%0Atranscoder

Reply:
dictionary.txt is not checked against aspell.

Reply:
LOL, shows what a n00b I am :wink: 
In that case, it's probably worth adding `tracsodes->transcodes` too.

Reply:
Yeah sadly there are too many words not in their dictionary to guarantee everything is present.

Added that.